### PR TITLE
Fix updating realtime table config

### PR DESCRIPTION
**Pull request by mcvsubbu**
Although the metadata was updated correctly, the PUT call returned an exception
because it found the kafka partition directory. We should not be checkig for the
kafka partition directory anymore

### CODE REVIEW

File: pinot-common/src/main/java/com/linkedin/pinot/common/metadata/ZKMetadataProvider.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import com.linkedin.pinot.common.utils.SchemaUtils;
 import com.linkedin.pinot.common.utils.SegmentName;
 import com.linkedin.pinot.common.utils.StringUtil;
+import com.sun.istack.internal.NotNull;
```
Comment (Jackie-Jiang) [inline pinot-common/src/main/java/com/linkedin/pinot/common/metadata/ZKMetadataProvider.java]: Wrong import

File: pinot-common/src/main/java/com/linkedin/pinot/common/metadata/ZKMetadataProvider.java [hunk 2]
```diff
@@ -323,7 +325,7 @@ public static Schema getTableSchema(@Nonnull ZkHelixPropertyStore<ZNRecord> prop
     return resultList;
   }
 
-  public static List<LLCRealtimeSegmentZKMetadata> getLLCRealtimeSegmentZKMetadataListForTable(
+  public static @NotNull List<LLCRealtimeSegmentZKMetadata> getLLCRealtimeSegmentZKMetadataListForTable(
```
Comment (Jackie-Jiang) [inline pinot-common/src/main/java/com/linkedin/pinot/common/metadata/ZKMetadataProvider.java]: We usually put @NonNull in front of the method declaration as a separate line (refer to https://www.programcreek.com/java-api-examples/?api=javax.annotation.Nonnull)

Comment (mcvsubbu) [reply pinot-common/src/main/java/com/linkedin/pinot/common/metadata/ZKMetadataProvider.java]: Done

**Comment by mcvsubbu** [comment]
Right now, the only way to test this is by adding integration test. I would prefer that we make pinotHelixResourceManager unit-testable instead of adding an integration for each of these cases, since it increases travis time.

**Pull request merged**